lib/TPP/TppCompiler.cpp

```diff
@@ -45,8 +45,8 @@ void TppCompilerPipeline::runOnOperation() {
   // map-linalg-to-tpp
   pm.addNestedPass<func::FuncOp>(createMapLinalgToTppPass());
   // enforce-tpp-preconditions
-  if (enablePreconditions)
-    pm.addNestedPass<func::FuncOp>(createPasSIMDDimensionPass());
+  // if (enablePreconditions)
```

Comment:
Don't comment out, remove the lines

Reply:
The entire pass was removed, thus here I commented out. There is not more this file in main. I will rebase it.

Reply:
My point was just about the commenting out vs removing the lines. Leaving code commented out all over the file is a mess and confuses people.